addon/components/query-syntax/template.hbs

```diff
@@ -0,0 +1,100 @@
+            <li><code>+giraffe neck spots -stripes</code> -- {{t 'eosf.components.querySyntax.booleanDesc6'}} "stripes" <b>{{t 'eosf.components.querySyntax.mustNot'}}</b> {{t 'eosf.components.querySyntax.booleanDesc7'}}.</li>
+            <li><code>giraffe AND lion</code> -- {{t 'eosf.components.querySyntax.both'}} "giraffe" {{t 'eosf.components.querySyntax.and'}} "lion" <b>must</b> {{t 'eosf.components.querySyntax.booleanDesc7'}}. {{t 'eosf.components.querySyntax.booleanDesc8'}} <code>+giraffe +lion</code>.</li>
+            <li><code>giraffe AND NOT lion</code> -- {{t 'eosf.components.querySyntax.booleanDesc9'}} "giraffe" <b>must</b> {{t 'eosf.components.querySyntax.booleanDesc7'}}, {{t 'eosf.components.querySyntax.theWord'}} "lion" <b>{{t 'eosf.components.querySyntax.mustNot'}}</b>. {{t 'eosf.components.querySyntax.booleanDesc8'}} <code>+giraffe -lion</code>.</li>
+            <li><code>tags:(ungulate OR feline)</code> -- {{t 'eosf.components.querySyntax.booleanDesc10'}} "ungulate", "feline", {{t 'eosf.components.querySyntax.or'}} {{t 'eosf.components.querySyntax.both'}}.</li>
+        </ul>
+    </div>
+    <div>
+        <h3>{{t 'eosf.components.querySyntax.wildcards'}}</h3>
+        <p>
+            {{t 'eosf.components.querySyntax.wildcardsDesc1'}} <code>?</code> {{t 'eosf.components.querySyntax.wildcardsDesc2'}} <code>*</code> {{t 'eosf.components.querySyntax.wildcardsDesc3'}}
+        </p>
+        <ul>
+            <li><code>giraf*</code> -- {{t 'eosf.components.querySyntax.matchWordStartsWith'}} "giraf".</li>
+            <li><code>l?on</code> -- {{t 'eosf.components.querySyntax.match'}} "lion", "loon", "leon", etc.</li>
+        </ul>
+    </div>
+    <div>
+        <h3>{{t 'eosf.components.querySyntax.fuzziness'}}</h3>
+        <p>
+            {{t 'eosf.components.querySyntax.use'}} <code>~</code> {{t 'eosf.components.querySyntax.fuzzinessDesc1'}}
+        </p>
+        <ul>
+            <li><code>girafe~ nekc~</code> -- {{t 'eosf.components.querySyntax.match'}} "giraffe neck".</li>
+        </ul>
+        <p>
+            {{t 'eosf.components.querySyntax.thisUsesThe'}} <a href='http://en.wikipedia.org/wiki/Damerau-Levenshtein_distance'>Damerau-Levenshtein distance</a> {{t 'eosf.components.querySyntax.damerauLevenshteinDistanceDesc'}} <code>~</code>.
+        </p>
+        <ul>
+            <li><code>girafe~2</code> -- {{t 'eosf.components.querySyntax.match'}} "giraffe", "gyrate", "pirate", etc.</li>
+        </ul>
+    </div>
+    <div>
+        <h3>{{t 'eosf.components.querySyntax.phraseProximity'}}</h3>
+        <p>
+            {{t 'eosf.components.querySyntax.phraseProximityDesc1'}}
+        </p>
+        <ul>
+            <li><code>"giraffe neck"~2</code> -- {{t 'eosf.components.querySyntax.match'}} "giraffe has a neck", {{t 'eosf.components.querySyntax.butNot'}} "giraffe has a long neck".</li>
```

Comment:
I chose not to translate these examples because I was concerned about them being translated properly.